tests/network-setups/fabric/dev/scripts/setupAppCCLocal.sh

```diff
@@ -0,0 +1,24 @@
+CC_NAME=${1:-"simplestate"}
+
+INTEROPCC_CHAINCODE_PATH=$PWD/../../../../core/network
+CHAINCODE_PATH=$PWD/../shared/chaincode
+APP_CC_PATH=$PWD/../../../../samples/fabric/${CC_NAME}$2
+
+echo "Copying ${CC_NAME}..."
+
+##################### Copying app chaincode ###############
+if [ -d "${CHAINCODE_PATH}/${CC_NAME}" ]; then
+    echo "Deleting existing ${CC_NAME} chaincode folder to copy the latest..."
+    rm -rf ${CHAINCODE_PATH}/${CC_NAME}
+fi
+cp -r ${APP_CC_PATH} ${CHAINCODE_PATH}/${CC_NAME}
+cp -r ${INTEROPCC_CHAINCODE_PATH}/fabric-interop-cc/libs/utils ${CHAINCODE_PATH}/${CC_NAME}/
```

Comment:
Is it really required to copy test related things into the chaincode? I think for deployment test related files were not required if `go.mod` and `go.sum` are already generated.

Reply:
`utils` isn't related only to tests. It's imported in the source files.